src/main/resources/compendium.yaml

```diff
@@ -0,0 +1,250 @@
+openapi: 3.0.0
+info:
+  description: Compendium is a service for schema storage, conversion, version management,
+    and client generation. It supports a RESTful interface for storing and
+    serving schemas like Avro, Protobuf, OpenAPI, and Mu.
+  version: 0.0.1
+  title: Compendium
+  contact:
+    email: contact@higherkindness.io
+  license:
+    name: Apache 2.0
+    url: http://www.apache.org/licenses/LICENSE-2.0.html
+externalDocs:
+  description: Learn more about Compendium
+  url: https://github.com/higherkindness/compendium
+servers:
+  - url: '{protocol}://{host}:{port}/{urlPrefix}'
+    variables:
+      protocol:
+        enum:
+          - http
+          - https
+        default: http
+      host:
+        default: localhost
+      port:
+        default: "8080"
+      urlPrefix:
+        default: v0
```

Comment:
Should it be in every endpoint instead? If we evolve the API in the future, the prefix will not be `/v0` for all endpoints, and the endpoints will not be the same for `v0` or `v1`.

We could even create a tag for `v0` version endpoints :thinking:

Reply:
It makes sense, `v0` should not be a default in the URL 😕